sysutils/exa/Portfile

```diff
@@ -4,110 +4,134 @@ PortSystem          1.0
 PortGroup           github 1.0
 PortGroup           cargo 1.0
 
-github.setup        ogham exa 0.9.0 v
-revision            1
+github.setup        ogham exa 0.10.1 v
+revision            0
 categories          sysutils
 platforms           darwin
 maintainers         {catlett.info:chad @chadcatlett} \
+                    {harens @harens} \
                     openmaintainer
 license             MIT
 
 description         a modern replacement for ls
-long_description    exa is ${description}
+long_description    ${name} is an improved file lister with more features \
+                    and better defaults. It uses colours to distinguish file types \
+                    and metadata. It knows about symlinks, extended attributes, and Git. \
+                    And it’s small, fast, and just one single binary.
+
+github.tarball_from archive
+
+homepage            https://the.exa.website/
 
 checksums           exa-${version}.tar.gz \
-                    rmd160  2633498619bf5ecef415aa92b8766d91a9fdffac \
-                    sha256  77fe070b73fdcd310720c733f9c32b0f81ad2958a2f8a38d3f0f85745e4b02d0 \
-                    size    560093
+                    rmd160  5809f633e71881eff721d2de34c1d0d8312d5f80 \
+                    sha256  ff0fa0bfc4edef8bdbbb3cabe6fdbd5481a71abbbcc2159f402dea515353ae7c \
+                    size    574807
+
+depends_lib         port:libiconv \
```

Comment:
Please use `depend_lib-append` here so as to not potentially wipe out any deps set by the port groups.

Reply:
Thanks for the review @cjones051073. I checked beforehand, and I didn't notice it removing any deps (since `cargo` is a build dep), but I changed it anyway just in case.